src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -166,7 +166,7 @@ class PaymentInformationEditModal extends React.Component {
 
           <LoadingButton
             type="submit"
-            className="usa-button-primary vads-u-width--auto"
+            className="usa-button-primary vads-u-width--full small-screen:vads-u-width--auto"
```

Comment:
Huge high five to @ncksllvn for telling me these responsive class prefixes exist!